contracts/hts-precompile/README.md

```diff
@@ -0,0 +1,68 @@
+## HTS Precompile methods
+
+The following table highlights the coded HTS precompile methods and their availability in the Hedera network exposed via the `0x167` address.
+
+| Function Name                             | hash          | Consensus Node Release Version | HIP                              | Method Interface | Comments
+|-------------------------------------------|---------------|--------------------------------|----------------------------------|------------------|---------------|
+| `allowance`                               | `0x927da105`  | [0.26](https://docs.hedera.com/hedera/networks/release-notes/services#v0.26)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `allowance(address,address,address)` |
+| `allowance`                               | `0xdd62ed3e`  | [0.26](https://docs.hedera.com/hedera/networks/release-notes/services#v0.26)  | [HIP 376](https://hips.hedera.com/hip/hip-376)    | `allowance(address,address,address)` | ERC allowance |
+| `approve`                                 | `0xe1f21c67`  | [0.26](https://docs.hedera.com/hedera/networks/release-notes/services#v0.26)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `approve(address token, address to, uint256 tokenId)` |
+| `approve`                                 | `0x95ea7b3`   | [0.26](https://docs.hedera.com/hedera/networks/release-notes/services#v0.26)  | [HIP 376](https://hips.hedera.com/hip/hip-376)    | `approve(address token, address to, uint256 tokenId)` | ERC approve |
```

Comment:
4byte.directory says that `95EA7B3` belongs to a _different_ signature `approve(address,uint256)` - https://www.4byte.directory/signatures/?bytes4_signature=95ea7b3

Reply:
(leads to a couple of questions: how did these tables get generated? and, if not by a script, could they be from a script?)

Reply:
@david-bakin-sl is right, the correct signature for `approve(address,address,uint256)` is `0xe1f21c67`.

Reply:
Thanks let me verify, lots of hashes and a few have v2's and v3's caused by protobuf changes or struct enhancements

Reply:
I've added a page for quick encoding, if anyone wants to additionally verify those hashes. [Link](https://georgi-l95.github.io/web3-playground/getEncodedFunctionSignature/)